frontend/src/components/Following/FollowingButton.js

```diff
@@ -0,0 +1,67 @@
+import '../../config';
+import React, { useState, useEffect } from 'react';
+import axios from "axios";
```

Comment:
Picky comment to adhere to the coding standards: would you mind moving the axios import up a line so the importants can be sorted alphabetically? 😅